### PR TITLE
Fix page sections vertical spacing

### PR DESCRIPTION
**Pull request by lilyndk**
- Add more sizes to bootstrap $spacers variables
- Increase page sections vertical spacing (home, mentors, resources, teams pages); remove hardcoded spacing values

Before (home):
spacing-home-before

After (home):
spacing-home-after

Before (mentors):
spacing-mentors-before

After (mentors):
spacing-mentors-after

Before (resources):
spacing-resources-before

After (resources):
spacing-resources-after

Before (teams):
spacing-teams-before

After (teams):
spacing-teams-after

- Increase mentorship tips card vertical spacing (resources page)

Before:
spacing-resources-card-before

After:
spacing-resources-card-after

## Type Of Change
- [ ] Update documentation
- [ ] Add feature
- [x] Fix 

## Pull request checklist

Please check if your PR fulfills the following requirements:
- [x] Commit message is as per guidelines in the [contributor guide](https://github.com/WomenWhoCode/london/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: mentors.html [hunk 1]
```diff
@@ -2,15 +2,15 @@
 layout: default
 title: Meet our community of mentors!
 ---
-<div class="mentors container">
+<div class="mentors container my-7">
```
Comment (dricazenck) [inline mentors.html:5]: Could you move this class to `.mentors` the same goes for the other changes, it is cleaner just change scss. 

in that case you can just do something like this: 

```
.mentors {
      @extend .my-7;
}
```

File: _sass/bootstrap/_variables.scss [hunk 2]
```diff
@@ -124,7 +124,9 @@ $spacers: map-merge((
   2: ($spacer * .5),
   3: $spacer,
   4: ($spacer * 1.5),
-  5: ($spacer * 3)
+  5: ($spacer * 3),
```
Comment (dricazenck) [inline _sass/bootstrap/_variables.scss:127]: 👍

File: mentors.html [hunk 3]
```diff
@@ -2,15 +2,15 @@
...
     <h1>{{page.title}}</h1>
 
     <div class="description">
         Below you can view all our mentors. The Application starts in February, 2023.
     </div>
 
     {% for mentor in site.data.mentors %}
-        <div class="card mb-3">
+        <div class="card mb-3 my-6">
```
Comment (dricazenck) [inline mentors.html:13]: In this case could be for example: 

```
.mentors {
    .card {
           @extend .mb-3;
           @extend .my-6;
    }
}
```

what do you think?

File: mentors.html (hunk 1 above)
Comment (lilyndk) [reply mentors.html]: Yes thanks, it's resolved now.

File: mentors.html (hunk 3 above)
Comment (lilyndk) [reply mentors.html]: Yes thanks for the suggestion. I overlooked that we don't need mb-3, so I've removed that entirely as well in the latest commit.

File: resources.html [hunk 4]
```diff
@@ -37,7 +37,7 @@ <h1>Mentorship Blog Posts</h1>
                             </div>
 
                             <div class="col-md-7 col-lg-8">
-                                <div class="card-body">
+                                <div class="card-body my-4">
```
Comment (dricazenck) [inline resources.html:40]:
```suggestion
                                <div class="card-body">
```

Comment (lilyndk) [reply resources.html]: Fixed in latest commit @dricazenck

**Review by dricazenck — APPROVED** [review]

**Pull request merged**